src/main/java/online/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -36,6 +39,9 @@ public UserResponseDto registration(UserRegistrationRequestDto requestDto) {
         user.setPassword(passwordEncoder.encode(requestDto.getPassword()));
         user.setRoles(Set.of(roleRepository.findByName(Role.RoleName.ROLE_USER)
                 .orElseThrow(() -> new EntityNotFoundException("Can't find role"))));
+        ShoppingCart cart = new ShoppingCart();
+        cart.setUser(user);
+        shoppingCartRepository.save(cart);
         return userMapper.toDto(userRepository.save(user));
```

Comment:
create method in shoppingCartService `createShoppingCart` and implement it 

```suggestion
        userRepository.save(user)
        shoppingCartService.createShoppingCart(user);
        return userMapper.toDto(user);
```